grid_map_pcl/package.xml

```diff
@@ -18,7 +18,6 @@
   <depend>grid_map_core</depend>
   <depend>grid_map_msgs</depend>
   <depend>grid_map_ros</depend>
-  <depend>pcl_ros</depend>
```

Comment:
You must then replace the rosdep keys to include PCL

Reply:
Forgive my ignorance, but which files should I edit to change the rosdep keys?

Reply:
in this package.xml file, you have to add depends so that when you use rosdep it can properly install all of the dependencies. It is also requires so the build farm will work properly. https://github.com/ros/rosdistro see keys here. Look at the docs / tutorials on using rosdep / rosdistro.

Reply:
I've updated the package.xml file to have the correct dependency on `libpcl-all`. When it comes to the rosdep / rosdistro repo, I don't see any rosdep keys for anything grid_map related in `base.yaml`... should there already be a key there?